README.md

```diff
@@ -13,23 +12,30 @@ Install `eslint` and `eslint-config-exp`:
 npm install --save-dev eslint eslint-config-exp
 ```
 
-Add the following to your `.eslintrc.json`-file:
+Then you need to add the following to your `.eslintrc.json`-file:
 
 ```json
 {
   "root": true,
-  "extends": ["eslint-config-exp"]
+  "extends": [ "exp" ]
 }
 ```
 
-Add the following to your `test/.eslintrc.json`-file:
+The configuration will look at the [type property](https://nodejs.org/api/packages.html#type) in your `package.json` to determine
+whether ES6 module rules should be applied or not. For Node versions that support it, the `es2022` environment will also be activated.
+
+You can also choose to use the test config, which is adapted to testing using `mocha`, `mocha-cakes-2` and `chai`. To also enable this,
+add it to your config `extends` like:
 
 ```json
 {
-  "extends": ["eslint-config-exp/test"]
+  "root": true,
+  "extends": [ "exp", "exp/test" ]
```

Comment:
please add a new target: all which contains both base and tests !

Reply:
Done!